test/index.js

```diff
@@ -34,17 +36,121 @@ describe('Wigzo', function () {
   describe('Constructing the integration', function () {
     it('should have the right settings', function () {
       analytics.compare(Wigzo, Integration('Wigzo')
+        .readyOnInitialize()
+        .global('wigzo')
+        .option('licenseCode', ''));
```

Comment:
what is this? this was not in `lib/index` also we should add the other options here `features` and `orgToken`